components/raftstore-v2/src/worker/pd/region.rs

```diff
@@ -311,12 +311,8 @@ where
                             );
                         }
                     } else if resp.has_merge() {
-                        PD_HEARTBEAT_COUNTER_VEC.with_label_values(&["merge"]).inc();
-
-                        let merge = resp.take_merge();
-                        info!(logger, "try to merge"; "region_id" => region_id, "merge" => ?merge);
-                        let req = new_merge_request(merge);
-                        send_admin_request(&logger, &router, region_id, epoch, peer, req, None);
+                        // TODO: add merge when it end-to-end works.
```

Comment:
But how to test end to end if this is removed entirely? I think it's better to use `merge_check_tick_interval` to control it. e.g. if it's larger than one hour, we implicitly ignore `PrepareMerge` in raftstore v2, and in optimize_for, we set v2 default to 1h.